pipeline/preprocessing.py

```diff
@@ -43,6 +43,7 @@
 output_mask = []
 
 
+# TODO this function is obsolete, remove in future
 def check_gradient(Nhdr_file):
```

Comment:
With my modification, some functions have become obsolete. We should remove them in future ...

Reply:
Sure, its better for release